src/libs/reportUtils.js

```diff
@@ -152,6 +152,15 @@ function isConciergeChatReport(report) {
         && report.participants[0] === CONST.EMAIL.CONCIERGE;
 }
 
+/**
+ * Returns true if there is any automated expensify account in emails
+ * @param {Array.<String>} emails
```

Comment:
I think we can just do `Array` here. Our JS docs are usually kept very simple.

Reply:
Done